docs/components/modeler/web-modeler/launch-web-modeler.md

```diff
@@ -1,29 +1,21 @@
 ---
 id: launch-web-modeler
 title: Launch Web Modeler
-description: "Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately. To launch Web Modeler, take these steps."
+description: "Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms."
 ---
 
 <span class="badge badge--cloud">Camunda 8 only</span>
 
-Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately.
+Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms.
 
 To launch Web Modeler, follow the steps below:
 
-1. Click the **Modeler** tab at the top of the page.
-   ![cloud web modeler menu item](img/cloud-web-modeler-menu-item.png)
-2. Click **New project** to create a new project to store all your diagrams.
-   ![web modeler empty home](img/web-modeler-new-user-new-project.png)
-   :::note
-   You can go back and rename your project at any time.
-   :::
-3. Click **New** and choose **Browse blueprints**. Alternatively, you can also create blank BPMN diagrams, DMN diagrams, or forms.
-   ![web modeler empty project](img/web-modeler-new-user-new-diagram-choose-browse-templates.png)
-4. From the modal, hover a tile and select an existing template (i.e. Absence Request) by clicking on the **Use template** button. Alternatively, you can create a blank diagram by clicking on the **Create a blank model** tile.
-   ![web modeler templates list modal](img/web-modeler-new-user-templates-list-modal.png)
-5. From the modal, you can also open the details of a specific template by clicking **Details**.
-   - In the details modal, you can find information about the author, the technical level, the description, and the categories of the selected template. You can also browse the diagram from the preview on the right side. To proceed with creating the diagram, click **Create model from template**.
-     ![web modeler template details modal](img/web-modeler-new-user-template-details-modal.png)
+1. After [creating an account and logging in](/guides/create-account.md) to Camunda, you are automatically taken to Web Modeler. Navigate between the components by clicking the square-shaped **Camunda components** icon in the top left corner of the page.
+2. Select **Create new project** to create a new project and store diagrams.
+   ![web modeler empty home](img/web-modeler-new-user-home.png)
+3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.
+4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagrams.
```

Comment:
Is it "blueprints" or "Blueprints" in the product? The casing should be consistent, but it's if inconsistent across the product let's be consistent **with** the product until that's changed.

Reply:
This is indeed how it is referenced in the product currently with the alternating casing -- happy to raise it with the Modeler team as far as UI goes, but matching it in the docs for now.

Reply:
Thank you! Please raise it with them.